src/client/components/DistrictOptionsFlyout.tsx

```diff
@@ -0,0 +1,49 @@
+/** @jsx jsx */
+import { jsx, Box } from "theme-ui";
+import { Button as MenuButton, Wrapper, Menu, MenuItem } from "react-aria-menubutton";
+import { DistrictId } from "../../shared/entities";
+import { style, invertStyles } from "./MenuButton.styles";
+import store from "../store";
+import { setZoomToDistrictId } from "../actions/districtDrawing";
+import { assertNever } from "../functions";
+
+type MenuKeys = "zoom-to-district";
+
+const DistrictOptionsFlyout = ({
+  districtId,
+  isDistrictHovered
+}: {
+  readonly districtId: DistrictId;
+  readonly isDistrictHovered: boolean;
+}) => {
+  return (
+    <Wrapper
+      onSelection={(menuKey: MenuKeys) => {
+        const action =
+          menuKey === "zoom-to-district" ? setZoomToDistrictId(districtId) : assertNever(menuKey);
+        store.dispatch(action);
+      }}
+      style={{ visibility: isDistrictHovered ? "visible" : "hidden" }}
+    >
+      <MenuButton
+        sx={{
+          ...{ variant: "icon", fontWeight: "light", color: "black", px: 1, py: 1 },
+          ...invertStyles({ invert: false })
+        }}
+      >
+        ...
```

Comment:
Using the middle dot character here will improve the styling just a little:
```suggestion
        ···
```